lib/feature/home_page/widgets/type.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex/utilities/extensions.dart';
+
+class Type extends StatelessWidget {
+  const Type(this.typeName);
+
+  final String? typeName;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          padding: EdgeInsets.symmetric(
+            horizontal: 12,
+            vertical: 5,
+          ),
+          margin: EdgeInsets.only(bottom: 8),
```

Comment:
Add **right** property since it will also be used horizontally in the details page